ktransformers/operators/experts.py

```diff
@@ -213,7 +213,7 @@ def load(self, w: dict | nn.Parameter | tuple | None = None, device:str|None = N
                 self.config.num_experts_per_tok,
                 self.config.hidden_size,
                 self.config.moe_intermediate_size,
-                max(cuda_graphs),
+                max(cuda_graphs) if isinstance(cuda_graphs, list) else 2048, # TODO: change the hard coded value
```

Comment:
The behavior of "--no-use_cuda_graph" mode is changed......
Also this is unrelated to XPU, if you prefer to change, do it in a separate patch with enough clarification for this magic number.

Reply:
I wonder what is the behavior of "--no-use_cuda_graph" mode now ?
Yes this is not only related to XPU. For XPU, now we don't have a cuda_graphs list, but we need to set a max_len value for AMX config otherwise the AMX support is broken (raise runtime error).
Actually this code may also bring issue for cuda local chat now. For local chat mode, the default chunk size is 8192, but for serve mode, the default chunk size is 256. This causes if we pass a long prompt (maybe 800-2000 tokens), I guess cuda chat mode will also meet AMX core dumped issue as input tensor length will exceed AMX's max length.
But well, I can remove this change in this PR and we can solve this issue in another PR and figure out if there is any better solution.

Reply:
Updated the default value from hard coded 2048 to `Config().chunk_size`, add also updated the value in local chat. Please let me know if this is more acceptable . 😊 @aubreyli @qiyuxinlin

Reply:
You didn't enable AMX in your yaml, I still prefer another PR for this change.

Reply:
Sure, have reverted related code changes, and maybe postponed until next PR.